packages/ag-charts-community/src/chart/chartProxy.ts

```diff
@@ -63,4 +120,32 @@ export class AgChartInstanceProxy implements AgChartProxy {
     destroy() {
         this.chart.destroy();
     }
+
+    private async prepareResizedChart({ chart }: AgChartInstanceProxy, opts: DownloadOptions = {}) {
+        const width: number = opts.width ?? chart.width ?? chart.ctx.scene.canvas.width;
+        const height: number = opts.height ?? chart.height ?? chart.ctx.scene.canvas.height;
+
+        const options: ChartExtendedOptions = mergeDefaults(
+            {
+                container: document.createElement('div'),
+                overrideDevicePixelRatio: 1,
+                width,
+                height,
+            },
+            // Disable enterprise features that may interfere with image generation.
+            moduleRegistry.hasEnterpriseModules() && { animation: { enabled: false } },
+            chart.userOptions
```

Comment:
Should this be `chart.getOptions()`?

Reply:
Good shout, this was a pre-existing bug waiting to happen!

Reply:
Fixed